feature/communicate/src/main/java/com/cybozu/sample/kintone/spaces/feature/communicate/thread/ThreadScreen.kt

```diff
@@ -96,29 +102,63 @@ fun ThreadContent(
             }
         } else {
             if (!uiState.isError) { // 正常に読み込まれていればリスト表示
-                println("debug point")
-                LazyColumn(
-                    modifier =
-                        Modifier
-                            .fillMaxSize()
-                            .padding(innerPadding),
-                    contentPadding = PaddingValues(all = 16.dp),
-                    verticalArrangement = Arrangement.spacedBy(8.dp)
-                ) {
-                    items(uiState.threadMessages) { threadMessage ->
-                        MessageListItem(threadMessage = threadMessage)
-                    }
-                }
-            } else { // 読み込めなければエラー表示
-                val context = LocalContext.current
-                LaunchedEffect(uiState.isError) {
-                    Toast.makeText(context, "メッセージを取得できませんでした", Toast.LENGTH_SHORT).show()
-                }
+
+                RefreshBox(
+                    items = uiState.threadMessages,
+                    isRefreshing = false,
+                    onRefresh = onRefresh,
+                    paddingValues = innerPadding,
+                    modifier = Modifier
+                )
+            } else {
+                ErrorMessage(
+                    context = LocalContext.current,
+                    isError = uiState.isError
```

Comment:
[必須]
IDEの警告も出てるかもですが、ここではuiState.isErrorは常にtrueなので以下のように書けそうですね。
```suggestion
                    isError = true
```